docs/changes/config/17461.perf.rst

```diff
@@ -0,0 +1 @@
+Optimize getting config item values.
```

Comment:
Can we roughly quantify this?

Reply:
It's not a noticeable change if you get a few config values but it is for `io.fits` which uses config items when e.g. getting card values. So when reading many keywords or a table with many columns it starts to matter.

Reply:
I didn't know FITS uses this... 🙈